concepticondata/conceptlists.tsv

```diff
@@ -278,3 +278,4 @@ Joo-2019-100	Joo, Ian	2019		100	ranked, basic	English	Global		Joo2019		This list
 Tolmie-1884-211	Tolmie, William F. and Dawson, George M.	1884		211	basic	English	Languages of the Native Tribes of British Columbia	https://open.library.ubc.ca/collections/bcbooks/items/1.0222394	Tolmie1884	Tolmie1884	This collection of 211 items, based on [Instructions for Research Relative to the Ethnology and Philology of America](:bib:Gibbs1863), was collected by the authors during expeditions in the British Columbia territory during the winters of 1875 and 1876. It contains basic vocabulary terms in addition to local flora and fauna terms. The core list of 211 items, also known as Dawson-Tolmie Vocabularies, is extended by appendices with extended vocabularies for specific languages in the area, but is not considered part of the main vocabulary list.		Tolmie-Dawson Vocabulary
 Winter-2016-300	Winter, Bodo	2016		300	ranked	English	English	https://github.com/bodowinter/iconicity_senses/tree/master/data	Winter2016		This list of 300 items was used to elicit modality exclusivity norms of selected verb concepts, for which participants provided perceptual strength ratings across five sensory modalities (i.e., hearing, taste, touch, smell, and vision).		
 Lynott-2013-400	Lynott, Dermot and Connell, Louise	2013		400	ranked	English	English	https://link.springer.com/article/10.3758/s13428-012-0267-0	Lynott2013		This list of 400 items was used to elicit modality exclusivity norms of selected noun concepts, for which participants provided perceptual strength ratings across five sensory modalities (i.e., hearing, taste, touch, smell, and vision).		
+Lewis-2016-499	Lewis, Molly L. and Frank, Michael C.	2016		499	ranked	English	English	http://www.sciencedirect.com/science/article/pii/S0010027716300919	Lewis2016		The dataset consists of &quot;499 English words from the MRC Psycholinguistic Database [Wilson 1988](:bib:Wilson1988) that were broadly distributed in their length and were relatively high frequency.&quot; (Lewis and Frank 2016: 191) &quot;with bootstrapped 95% CIs&quot; see [README norms.md](https://github.com/mllewis/RC/blob/master/data/norms/README%20-%20norms.md)		
```

Comment:
sorry for being picky here, but a dot in the end of the description is missing.

Reply:
Done ^^